filter_test.go

```diff
@@ -10,14 +10,14 @@ import (
 func TestFilterService_GetList(t *testing.T) {
 	setup()
 	defer teardown()
-	testAPIEdpoint := "/rest/api/2/filter"
```

Comment:
fixed typo.
(I found this typo in other test codes, but I fixed only this test codes.)